siliconcompiler/schema/schema_cfg.py

```diff
@@ -3774,39 +3770,24 @@ def schema_constraint(cfg):
             (horizontal, vertical) tuple represented in microns.""")
 
     scparam(cfg, ['constraint', 'component', inst, 'rotation'],
-            sctype='float',
+            sctype='enum',
```

Comment:
this will break the floorplanning setup, so we will need to fix that before merging.

Reply:
How come none of the CI tests are failing?

Reply:
This is part of the daily CI, so the basic checks do no test this